Rug.Domain/Match/Match.cs

```diff
@@ -1,4 +1,30 @@
-﻿namespace Rug.Domain.Match
+﻿using System.Collections.Generic;
+
+namespace Rug.Domain.Match
 {
-    public class Match {}
+    public class Match
+    {
+        public Match(Team.Team home, Team.Team away)
+        {
+            Home = home;
+            Away = away;
+        }
+
+        public Team.Team Home { get; }
+
+        public Team.Team Away { get; }
+    }
+
```

Comment:
Do you think this is the right place for the comparer (I mean in the domain) or we should move it elsewhere?

Reply:
its ok, until different comparison is needed. 
Only one question is:
Are matches between teams A and B should be treated as equal, if they happen on different hall or time or championship

Reply:
Yes, same participants can have different matches (in fact it should be like that), once played matches should not be replayed, there is a procedure of replaying a match but this is so rare that it hasn't happened in last 5-10 years, so we can leave it for later stage.